src/v2/Apps/Order/Routes/__tests__/Offer.jest.tsx

```diff
@@ -24,6 +24,11 @@ jest.mock("v2/Utils/Events", () => ({
   postEvent: jest.fn(),
 }))
 
+jest.mock("v2/Utils/user", () => ({
```

Comment:
should we remove this user mock since we will write the test case after retiring the feature flag?

Reply:
Thanks for the review!
without it some tests will break 🙃